.github/workflows/pre-submit.actions.yml

```diff
@@ -319,3 +320,79 @@ jobs:
           directory: "internal/builders/generic"
           # NOTE: compile-builder explicitly set to false.
           compile-builder: false
+
+  generate-attestations:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Create test inputs
+        run: |
+          cat <<EOF >DATA
+          {
+            "version": 1,
+            "attestations": [
+              {
+                "name": "attestation1.intoto",
+                "subjects": [
+                  {
+                    "name": "artifact11",
+                    "digest": { "sha256": "deadbeaf" }
+                  }
+                ]
+              },
+              {
+                "name": "attestation2.intoto",
+                "subjects": [
+                  {
+                    "name": "artifact21",
+                    "digest": { "sha256": "deadbeat" }
+                  }
+                ]
+              }
+            ]
+          }
+          EOF
+          cat DATA > slsa-outputs.json
+
+          cat <<EOF >PREDICATE
+          {
+            "buildType": "https://github.com/slsa-framework/slsa-github-generator/go@v1",
+            "builder": {
+              "id": "https://github.com/slsa-framework/slsa-github-generator/.github/workflows/builder_go_slsa3.yml@refs/tags/v0.0.1"
+            }
+          }
+          EOF
+          cat PREDICATE > predicate.json
+      # TODO(asraa): Enable generate-attestation action at main
```

Comment:
create a tracking issue if that helps not forget

Reply:
i removed, realized i can checkout and invoke as a local action :P